lib/testmap.py

```diff
@@ -288,7 +288,7 @@ def tests_for_image(image):
             for context in contexts:
                 if context.split('/')[0].replace('-distropkg', '') == image:
                     c = context + '@' + repo
-                    if branch != "master":
+                    if branch not in ["master", "main"]:
```

Comment:
This is fine. But it would be nicer if we would have something like:
```py
if branch != get_default_branch(repo):
    c += "/" + branch
```

and `get_default_branch()` can be as simple as:
```py
def get_default_branch(repo):
    if repo in ["cockpit-project/cockpit-machines"]: # Not sure where is main right now
        return "main"
    return "master"
```

Reply:
Good idea, I'll check that; I'm not sure if I'm a big fan of this explicit list, as opposed to checking for either "main" or "master", or even marking it somehow in the project list -- but either way, a `get_default_branch()` makes sense.